specification/network/resource-manager/Microsoft.Network/stable/2018-08-01/examples/AvailableDelegationsSubscriptionGet.json

```diff
@@ -0,0 +1,23 @@
+{
+  "parameters" : {
+    "api-version": "2018-08-01",
+    "subscriptionId" : "subId"
+  },
+  "responses" : {
+    "200" : {
+      "body" : {
+        "value": [

```

Comment:
In the spec, the array items are defined under "delegations" instead of "value".

Reply:
changed to value